scripts/jest/jest-cli.js

```diff
@@ -314,6 +329,10 @@ function getEnvars() {
     envars.VARIANT = true;
   }
 
+  if (argv.reactVersion) {
+    envars.REACT_VERSION = semver.minVersion(argv.reactVersion).version;
```

Comment:
I wonder if you considered using `coerce` rather than `minVersion`?

Reply:
😮  How did I miss that in the docs?!